configuration/rules-dsl.md

```diff
@@ -241,6 +241,10 @@ In openHAB version 3 the System-based Trigger for startlevel had been added, val
 
 Startlevels less than 40 are not available as triggers because the rule engine needs to start up first before it can execute any rules.
 
+A rule triggering for a startlevel and being saved during development, will be reloaded.
```

Comment:
A rule that triggers for a start level and is saved during development is reloaded.